compiler/one-cmds/one-infer

```diff
@@ -206,6 +209,16 @@ def main():
     # run backend driver
     _utils._run(infer_cmd, err_prefix=ntpath.basename(driver_path))
 
+    # run post process script if it's given
+    if _utils._is_valid_attr(args, 'post_process'):
+        # Currently, it is only supported to use one-venv.
+        # TODO: support local python env or specify python path
```

Comment:
I'm not good at python venv but technically is this possible?

Reply:
https://github.com/Samsung/ONE/blob/75a30387f36ebbceafe509e755e97687e82d8c95/compiler/one-cmds/one-infer#L216-L217
Yes if the `python_path` is changed to user's python path, it can be run by local python or another venv that user specified. (Even if it is set to `python3`, local python binary will be run)